src/App.js

```diff
@@ -245,9 +246,29 @@ export const App = ({
     }
   }, [pathname])
 
+  useEffect(() => {
+    const tableRowsCountChangeHandler = ({ detail }) =>
+      setIsLessThan20(detail < 20)
+    window.addEventListener(
+      'tableRowsCountChanged',
+      tableRowsCountChangeHandler,
+      false
+    )
+    return () =>
+      window.removeEventListener(
+        'tableRowsCountChanged',
+        tableRowsCountChangeHandler,
+        false
+      )
+  }, [])
+
   return (
     <div
-      className={cx('App', isWatchlistPage && isDesktop && 'list-container')}
+      className={cx(
+        'App',
+        isWatchlistPage && isDesktop && 'list-container',
+        isWatchlistPage && isDesktop && isLessThan20 && 'list-full-height'
```

Comment:
To reuse `isWatchlistPage && isDesktop` we can save it to a variable like `isDesktopWatchlistPage`